fetch.bs

```diff
@@ -175,8 +175,34 @@ lt="authentication entry">authentication entries</a> (for HTTP authentication).
 
  <dt><dfn for="fetch params">task destination</dfn> (default null)
  <dd>Null, a <a for=/>global object</a>, or a <a for=/>parallel queue</a>.
+ <dt><dfn for="fetch params">timing info</dfn>
+ <dd>A <a for=/>fetch timing info</a>.
 </dl>
 
+<p>A <dfn>fetch timing info</dfn> is a <a for=/>struct</a> used to maintain timing information
+later required by the resource timing and navigation timiing specs. It has the following <a for=struct>items</a>:
```

Comment:
s/timiing/timing/
Also It might be worthwhile to move the background info to a note

Reply:
OK